### PR TITLE
Upping to CodeMirror 5.44.0

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @devoncarew

**Comment by domesticmouse** [comment]
Looks like Travis died on `slash_for_doc_comments`. I'll update this PR with a fix for the warnings after presenting, or more likely, tomorrow. Please let me know if you see anything else out of place.

**Comment by domesticmouse** [comment]
Now with passing tests!

File: lib/panel.dart [hunk 1]
```diff
@@ -33,13 +29,9 @@ class Panel {
 class PanelContainer extends ProxyHolder {
   PanelContainer._(JsObject jsProxy) : super(jsProxy);
 
-  /**
-   * Used to remove the panel.
-   */
+  ///Used to remove the panel.
```
Comment (devoncarew) [inline lib/panel.dart]: needs a space

File: lib/panel.dart [hunk 2]
```diff
@@ -33,13 +29,9 @@ class Panel {
...
   void clear() => call('clear');
 
-  /**
-   * Used to notify the addon when the size of the panel's DOM node has changed.
-   */
+  ///Used to notify the addon when the size of the panel's DOM node has changed.
```
Comment (devoncarew) [inline lib/panel.dart]: needs a space

File: pubspec.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.0+5.43.0
+version: 0.5.0+5.44.0
```
Comment (devoncarew) [inline pubspec.yaml]: `0.5.1` - lets bump the patch version as well

Comment (domesticmouse) [reply pubspec.yaml]: Done

**Comment by domesticmouse** [comment]
Version bumped, changelog appended, spaces added.

**Pull request merged**